.plzconfig

```diff
@@ -6,29 +6,38 @@ PreloadBuildDefs = test/build_defs/test.build_defs
 
 [Go]
 ImportPath = github.com/please-build/python-rules
+; embedtool = @self//tools/please_go_embed
+gotool = @self//third_party/go:toolchain|go
 
+[Java]
+JarCatTool = @self//tools:arcat
 
 [PluginDefinition]
 name = python
 
 [PluginConfig "default_interpreter"]
 ConfigKey = DefaultInterpreter
 DefaultValue = python3
+Optional = true
```

Comment:
Do we really want this to be optional? That means I could do 
```
[Proto "python"]
DefaultInterpreter = 
```

And `CONFIG.PYTHON.DEFAULT_INTERPRETER` would be unset. This seems like it would break things.

Reply:
Yeah good point! I think actually most of them shouldn't be optional. I've changed them.